storage/remote/queue_manager.go

```diff
@@ -227,11 +242,11 @@ func (t *StorageQueueManager) sendSamples(s model.Samples) {
 	err := t.tsdb.Store(s)
 	duration := time.Since(begin).Seconds()
 
-	labelValue := success
 	if err != nil {
 		log.Warnf("error sending %d samples to remote storage: %s", len(s), err)
-		labelValue = failure
+		sentSamplesTotal.WithLabelValues(t.queueName).Add(float64(len(s)))
```

Comment:
This should increment `failedSamplesTotal` and below, it should increment `sentSamplesTotal`.